main.tex

```diff
@@ -14,6 +14,9 @@
 \usepackage[style=alphabetic,minalphanames=3,maxbibnames=99]{biblatex}
 \usepackage{hyperref}
 \usepackage{cleveref}
+\crefname{section}{\S}{\S\S}
+\Crefname{section}{\S}{\S\S}
```

Comment:
Not sure what this wants to be, but I have a hunch that you don't want to duplicate this line with different capitalization of "c"?

Reply:
there's separate commands for the middle and beginning of sentences, and in this case i'm overriding "section", "sections", "Section" and "Sections" to both be § and §§. \Cref is not used, so technically it's not needed but if we move things around this way it'll stay consistent